UPGRADE.md

```diff
@@ -181,28 +144,33 @@ autodetect Commands in the `Command` directory of each bundle anyway. This only
 affects you, if you have not followed the Symfony standards and located your
 commands somewhere else.
 
-### Preview
-
-Clear the preview cache to avoid wrong cached data.
-
-```bash
-app/console cache:clear
-```
-
 ### WebsiteRequestAnalyzer
 
 The `Current`-part of all setters have been removed, because they have already
 been removed from the getters. This only affects you if you have overridden the
 `WebsiteRequestAnalyzer` and have called or overridden these methods.
 
-### Webspace Settings
+### Databases
 
-A new phpcr namespace was added. To register it run following command:
+#### PHPCR
+
+A new namespace and additional system nodes were added. To create them run the following command:
 
 ```bash
-app/console sulu:phpcr:init
+app/console sulu:document:initialize
+```
+
+#### ORM
+
+The relational structure of categories, translations and users have changed. 
+Use the following command to update:
+
+```bash
+app/console doctrine:schema:update --force
 ```
 
+It might be possible that foreign key checks have to be disabled for this update.
```

Comment:
As @chirimoya pointed out this did not always happen. That's why I thought we should leave it like that. Also it might be different for different databases.